tests/test_helpers/test_export_mixins.py

```diff
@@ -155,3 +207,30 @@ def test_list_with_csv_download_for_no_tenant(export_csv_mixin):  # pylint: disa
     export_csv_mixin.request.query_params = {'download': 'csv'}
     response = export_csv_mixin.list(export_csv_mixin.request)
     assert response.status_code == http_status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.django_db
+def test_list_with_csv_download_for_tasks_limit(export_csv_mixin, user, tenant):  # pylint: disable=redefined-outer-name
+    """Test the list method for tasks limit per user."""
+    filename = 'text.csv'
+    related_id = 'test-id-1'
+    current_view = 'test_export'
+    export_csv_mixin.request.query_params = {'download': 'csv'}
+    DataExportTask.objects.create(
+        filename=filename, view_name=current_view, user=user, tenant=tenant,
+        related_id=related_id, progress=0.0, status=DataExportTask.STATUS_PROCESSING
+    )
+    DataExportTask.objects.create(
+        filename=filename, view_name=current_view, user=user, tenant=tenant,
+        related_id=related_id, progress=0.6, status=DataExportTask.STATUS_IN_QUEUE
+    )
+    DataExportTask.objects.create(
+        filename=filename, view_name=current_view, user=user, tenant=tenant,
+        related_id=related_id, progress=0.6, status=DataExportTask.STATUS_IN_QUEUE
+    )
+    DataExportTask.objects.create(
+        filename=filename, view_name=current_view, user=user, tenant=tenant,
+        related_id=related_id, progress=0.6, status=DataExportTask.STATUS_IN_QUEUE
```

Comment:
one of them is completed
```suggestion
        related_id=related_id, progress=0.6, status=DataExportTask.STATUS_COMPLETED
```